include/config/config_camera.h

```diff
@@ -58,7 +58,7 @@
 /**
  * Enables Puppy Camera 2, a rewritten camera that can be freely configured and modified.
  */
-// #define PUPPYCAM
+//#define PUPPYCAM
```

Comment:
Probably should have been a bit more clear; the commented defines in the other files do use a space here. Space is also better because it's consistent with VSCode's commenting shortcut (Ctrl + /).